WordPress/src/main/res/layout/reader_cardview_xpost.xml

```diff
@@ -0,0 +1,78 @@
+    card_view:cardBackgroundColor="@color/grey_lighten_30"
+    card_view:cardCornerRadius="@dimen/cardview_default_radius"
+    card_view:cardElevation="0dp">
+
+    <LinearLayout
+        android:id="@+id/layout_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?android:selectableItemBackground"
+        android:orientation="horizontal"
+        android:paddingBottom="@dimen/margin_large"
+        android:paddingLeft="@dimen/reader_card_content_padding"
+        android:paddingRight="@dimen/reader_card_content_padding"
+        android:paddingTop="@dimen/margin_large">
+
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical">
+
+            <org.wordpress.android.widgets.WPNetworkImageView
+                android:id="@+id/image_blavatar"
+                android:layout_width="@dimen/blavatar_sz"
+                android:layout_height="@dimen/blavatar_sz"
+                android:layout_gravity="top|left"
+                tools:src="@drawable/blavatar_placeholder" />
+
+            <org.wordpress.android.widgets.WPNetworkImageView
+                android:id="@+id/image_avatar"
+                style="@style/ReaderImageView.Avatar.Small"
+                android:layout_gravity="bottom|right"
+                android:layout_marginLeft="14dp"
+                android:layout_marginTop="14dp"
+                tools:src="@drawable/gravatar_placeholder" />
+        </FrameLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginLeft="@dimen/margin_large"
+            android:orientation="vertical">
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/text_title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:maxLines="1"
+                android:textColor="@color/grey_dark"
+                android:textSize="@dimen/text_sz_medium"
+                android:textStyle="bold"
+                tools:text="text_title"
+                wp:fontFamily="merriweather" />
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/text_subtitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:maxLines="2"
+                android:textColor="@color/grey"
+                android:textSize="@dimen/text_sz_small"
+                tools:text="text_subtitle" />
+
+        </LinearLayout>
+    </LinearLayout>
+</android.support.v7.widget.CardView>
```

Comment:
The web reader seems to align the title of the post to the top of the gravatar. I think it'd be good to do the same here. 

If there's one line of text in the cross-post, the title is too low:
<img width="240" alt="screen shot 2015-12-16 at 1 51 16 pm" src="https://cloud.githubusercontent.com/assets/789137/11854989/45b1ea28-a3fc-11e5-9d33-b000ac3f92f8.png">

And if there's more than one line, it is too high:
<img width="240" alt="screen shot 2015-12-16 at 1 51 36 pm" src="https://cloud.githubusercontent.com/assets/789137/11854997/51d4dbda-a3fc-11e5-99f3-f9133e989c2c.png">

I think you could just take out the vertical centering to fix it.